tests/jballerina-unit-test/src/test/resources/test-src/expressions/lambda/iterable/basic-iterable-with-variable-mutability.bal

```diff
@@ -64,81 +64,75 @@ function testBasicArray2(string[] values) returns string {
-//                                  return false;
-//                            });
-//    return [count, values];
-//}
-//
-//function testBasicMap2() returns [int, string, string[]] {
-//    int count = 0;
-//    string str = "start";
-//    map<string> m = {a:"A", b:"B", c:"C", d:"D", e:"E"};
-//    string[] values = m.entries().'map(function ([string, string] tuple) returns [string, string] {
-//                                var [key, value] = tuple;
-//                                str = str + "-"  + key + " : " + value;
-//                                count = count + 10;
-//                                return [key, value];
-//                            })
-//                      .filter(function ([string, string] v) returns boolean {
-//                                  var [k, t] = v;
-//                                  if (k == "a" || k == "e") {
-//                                      count = count + 10;
-//                                      return true;
-//                                  }
-//                                  return false;
-//                              })
-//                      .'map(function ([string, string] v) returns string {
-//                               var [v1, v2] = v;
-//                               count = count + 10;
-//                               return v1 + v2;
-//                           });
-//    str = str + "-end";
-//    count = count + 1;
-//    return [count, str, values];
-//}
-
-// XML WITH NESTED ITERABLE OPERATIONS
-// Uncomment after implementing xml.map langlib function.
-//
-//function xmlTest() returns [string, map<any>] {
-//    string str = "start";
-//    xml xdata = xml `<p:person xmlns:p="foo" xmlns:q="bar">
-//        <p:name>bob</p:name>
-//        <p:address>
-//            <p:city>NY</p:city>
-//            <q:country>US</q:country>
-//        </p:address>
-//        <q:ID>1131313</q:ID>
-//    </p:person>`;
-//
-//    int index = 0;
-//    map<xml> m = xdata.*.elements()[1].*.elements()
-//                 .'map(function (xml|string x) returns [string, xml] {
-//                            index += 1;
-//                            if x is xml {
-//                                str = str + "-" + string.convert(index);
-//                                return [string.convert(index), x];
-//                            }
-//                            return ["", xml ` `];
-//                      });
-//    str = str + "-end";
-//    return [str, m];
-//}
+function testBasicMap1() returns [int, string[]] {
+    map<string> m = {a:"A", b:"B", c:"C", d:"D", e:"E"};
+    int count = 0;
+    string[] values = m.keys().'map(function (string key) returns string {
+                                count = count + 10;
+                                var k = key;
+                                return k.toLowerAscii();
+                            })
+                      .filter(function (string v) returns boolean {
```

Comment:
It is better if this line is aligned with the `.keys`.

@NipunaRanasinghe Please check whether this formatting change can be supported via the IDEA(and LS) in the future.

Reply:
@Shan1024 +1. To be consistent, its better if we can try to indent w.r.t. the beginning of the RHS of the assignment statement, as below.
```
function foo() {
    var x = variableX.foo().bar()
                .match()
                .filter();
}
```

@NipunaMarcus  WDYT? Do we have already have predefined rules when formatting the assignment statements, in our formatting guide?,

Reply:
@NipunaRanasinghe yes we already have style guide rules on how to format assignment and how to format line breaks